openmdao/jacobians/jacobian.py

```diff
@@ -90,13 +90,6 @@ def _abs_key2shape(self, abs_key):
         system = self._system()
         abs2meta = system._var_allprocs_abs2meta
         of, wrt = abs_key
-        if system.comm.size > 1:
```

Comment:
Interesting. Just tested this change on my branch with distributed constraints, and tests all pass so guess it was not needed.